scripts/go-tool.sh

```diff
@@ -64,10 +64,16 @@ fi
 function invoke_go() {
   tool="$1"
   shift
+  # TODO(ROX-27054): Remove the redundant strictfipsruntime option if one is found to be so
+  if [[ -z "$GOTAGS" ]]; then
+    GOTAGS="$GOTAGS,strictfipsruntime"
+  else
+    GOTAGS="strictfipsruntime"
+  fi
   if [[ "$RACE" == "true" ]]; then
-    CGO_ENABLED=1 go "$tool" -race -ldflags="${ldflags[*]}" -tags "$(tr , ' ' <<<"$GOTAGS")" "$@"
+    GOEXPERIMENT=strictfipsruntime CGO_ENABLED=1 go "$tool" -race -ldflags="${ldflags[*]}" -tags "$(tr , ' ' <<<"$GOTAGS")" "$@"
   else
-    go "$tool" -ldflags="${ldflags[*]}" -tags "$(tr , ' ' <<<"$GOTAGS")" "$@"
+    GOEXPERIMENT=strictfipsruntime go "$tool" -ldflags="${ldflags[*]}" -tags "$(tr , ' ' <<<"$GOTAGS")" "$@"
   fi
 }
 
```

Comment:
The changes here would cover all but Scanner V4 which does things in its own manner.
You'd need to inject values there additionally
https://github.com/stackrox/stackrox/blob/91ab7138ebfa14692c78b17969d4a37f651abc3e/scanner/Makefile#L59-L65